internal/plugin_test.go

```diff
@@ -426,6 +426,111 @@ spec:
 	assertEqual(t, output, expected)
 }
 
+func TestCreatePolicyWithAnnotations(t *testing.T) {
+	t.Parallel()
+	tmpDir := t.TempDir()
+	createConfigMap(t, tmpDir, "configmap.yaml")
+	p := Plugin{}
+	p.PolicyDefaults.Namespace = "my-policies"
+	p.PolicyDefaults.PolicyAnnotations = map[string]string{"test-default-annotation": "default"}
+
+	policyConf := types.PolicyConfig{
+		Name: "policy-app-config",
+		Manifests: []types.Manifest{
+			{Path: path.Join(tmpDir, "configmap.yaml")},
+		},
+	}
+	p.Policies = append(p.Policies, policyConf)
```

Comment:
Could you please add two more policies? The first one would override the default annotations with some other annotations. The other would override the default annotations with an empty map.

Reply:
add test case for empty map override.